tests/brevitas_ort/test_quant_module.py

```diff
@@ -0,0 +1,37 @@
+from operator import mul
+from functools import reduce
+
+import torch
+import pytest
+
+from .common import *
+from pytest_cases import parametrize_with_cases
+from .quant_module_cases import QuantWBIOLCases
+from pytest_cases import parametrize_with_cases, get_case_id
+from tests.marker import requires_pt_ge
+
+@parametrize_with_cases('model', cases=QuantWBIOLCases)
+@pytest.mark.parametrize('export_type', ['qcdq', 'qop'])
+@requires_pt_ge('1.8.1')
```

Comment:
@volcacius To support bitwidth<8 and narrow_range quantization (enabled by default in weight quantization), we need Clip operation, which has been introduced only from pytorch 1.8.1. Should we raise an error like we do for the axis flag in per_channel quantization?